docker-testing/docker-compose-mysql.yml

```diff
@@ -1,7 +1,7 @@
 version: '2.4'
 services:
   mysql:
-    image: "mysql/mysql-server:5.7.12"
```

Comment:
We don't support MySQL 5 any more, we only support MySQL 8.0.1. Since thats where customer's Mattermost installation, and thus the Boards plugin would run, we should be making this change.

Reply:
We should also update the postgres image to version 11 in `docker-compose-postgres.yml` for the same reason.

Reply:
One second, I didn't realise this is the repo for standalone version of Boards. So as such, this change can be skipped.

Reply:
mysql-server version 5.*.* is deprecated and support is disabled by default. 

When I tried to run the server test case it gave this error:

Docker Image Format v1 and Docker Image manifest version 2, schema 1 support is disabled by default and will be removed in an upcoming release. Suggest the author of docker.io/mysql/mysql-server:5.7.12 to upgrade the image to the OCI Format or Docker Image manifest v2, schema 2. More information at https://docs.docker.com/go/deprecated-image-specs/

Reply:
@Rajat-Dabade You can probably make the change in your local to get it running but we probably want to avoid upgrading the database version in the repo without any previous tests and/or possible updates which are out of scope for this.

Reply:
Okay. got it.